Sources/Rendering/Core/Mapper/index.js

```diff
@@ -413,7 +413,11 @@ function vtkMapper(publicAPI, model) {
     return true;
   };
 
-  publicAPI.canUseTextureMapForColoring = (input) => {
+  publicAPI.canUseTextureMapForColoring = (scalars, cellFlag) => {
+    if (cellFlag) {
+      return true; // cell data always use textures.
+    }
```

Comment:
@bruyeret What about the case where `model.colorMode === ColorMode.DIRECT_SCALARS`? Should we move that condition to before this check? There is existing code using DIRECT_SCALARS (using cell field data) that break with this change.

Reply:
We need to keep this condition before the `interpolateScalarsBeforeMapping` check, as it shouldn't affect cells
But I can do:
```js
    if (cellFlag && !(model.colorMode === ColorMode.DIRECT_SCALARS)) {
      return true; // cell data always use textures.
    }
```
I can add a commit for that in PR #3095